specs/eip4844/beacon-chain.md

```diff
@@ -167,10 +167,10 @@ See [the full details of `blob_versioned_hashes` offset calculation](https://gis
 def tx_peek_blob_versioned_hashes(opaque_tx: Transaction) -> Sequence[VersionedHash]:
     assert opaque_tx[0] == BLOB_TX_TYPE
     message_offset = 1 + uint32.decode_bytes(opaque_tx[1:5])
-    # field offset: 32 + 8 + 32 + 32 + 8 + 4 + 32 + 4 + 4 = 156
+    # field offset: 32 + 8 + 32 + 32 + 8 + 4 + 32 + 4 + 32 + 4 = 188
```

Comment:
missing 4 bytes here I think: should be 192.   Let some comments in https://gist.github.com/protolambda/23bd106b66f6d4bb854ce46044aa3ca3